pkg/reconciler/names.go

```diff
@@ -25,3 +28,7 @@ func GetK8sServiceFullname(name string, namespace string) string {
 func GetServingK8SServiceNameForObj(name string) string {
 	return name + "-service"
 }
+
+func GetServingK8SRevNameForObj(name string) string {
```

Comment:
Please rename this to `GetServingRevisionNameForK8sService` to correctly indicate which conversion is supported here.